PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/LoadJsonConfigFile.java

```diff
@@ -5,7 +5,16 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.util.ArrayList;
```

Comment:
I think the LoadJsonFile needs improvement in terms of design and datastructure.
Let's do a follow up PR with redesign for this part.
Few considerations:
1. loadJsonObject -> rename loadAllRules (or something like that)
2. The return type of loadJsonObject should be a model class that represents a list of rule structure.
Something like:
`public RuleConfig {
    private List<String> methodsToCheck;
    private String registerMessage
etc
}`
3. Update the methods to be more contextual and relevant to the rule structure.